packages/itwinui-react/src/core/Buttons/IconButton/IconButton.tsx

```diff
@@ -18,6 +18,10 @@ export type IconButtonProps = {
    * Name of the button, shown in a tooltip and exposed to assistive technologies.
    */
   label?: React.ReactNode;
+  /**
+   * Passes props to IconButton icon.
+   */
+  iconProps?: React.ComponentProps<'span'>;
 } & Omit<ButtonProps, 'startIcon' | 'endIcon'>;
```

Comment:
will need to omit all the new props that were added to `Button`

Reply:
So I need to remove all the props that I added in this PR?

Reply:
not remove, just omit

```diff
- } & Omit<ButtonProps, 'startIcon' | 'endIcon'>;
+ } & Omit<ButtonProps, 'startIcon' | 'endIcon' | 'startIconProps' | 'endIconProps' | 'labelProps'>;
```

Reply:
Ok, updated